covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -84,19 +92,30 @@ def _create_graphs(
         return country_graphs
 
     @staticmethod
-    def _create_buttons(dashboard_type: DashboardType):
-        buttons = [
-            dcc.Dropdown(
-                id="prediction-event",
-                options=[
-                    dict(label=(event.date + timedelta(days=1)).strftime("%B %d"), value=event.name)
-                    for event in prediction_db.get_prediction_events()
-                    if event != OTHER
-                ],
-                value=BK_20200411.name,
-                style={"width": "220px", "margin": "8px 0"},
+    def _create_buttons(
+        dashboard_type: DashboardType, report_by_short_name: Dict[str, CountryReport]
+    ):
+        buttons = []
+        if (
+            dashboard_type == DashboardType.SingleCountry
+            or dashboard_type == DashboardType.AllCountries
+        ):
+            buttons.append(
+                dcc.Dropdown(
+                    id="prediction-event",
+                    options=[
+                        dict(
+                            label=(event.date + timedelta(days=1)).strftime("%B %d"),
```

Comment:
I feel like we've been talking about adding this field since 2018, way before COVID-19 :grinning: 
```suggestion
                            label=(event.creation_date).strftime("%B %d"),
```

Reply:
Good idea, I'll do it. In a short next PR :)